base/applications/mspaint/selectionmodel.cpp

```diff
@@ -18,6 +18,7 @@ SelectionModel::SelectionModel()
     , m_hbmMask(NULL)
     , m_ptStack(NULL)
     , m_iPtSP(0)
+    , m_bContentsChanged(FALSE)
     , m_bShow(FALSE)
```

Comment:
It is recommended to initialize within initializer list in the same order as the attributes are declared within the header.

Reply:
Fixed. Thanks.